### DISCUSSION TITLE
How to bypass serverless/vercel payload limit 4.5mb; using @payloadcms/storage-s3, payload v3 beta 108

### DISCUSSION THREAD

**Discussion by drewlonious** [payloadcms/payload#8594] [Q&A] [answered]
Anyone have any idea on how to bypass this? Would appreciate the help. 

POST error 413 (Payload Too Large)

**Comment by AndKenneth** [comment]
I'm getting this as well, assuming we will need some way for the browser to get a signed URL for upload direct to S3/Vercel Blob/ect instead of proxying through the serverless function.

**Comment by denolfe** [comment, marked as answer]
This capability is now available in v3.25.0 https://github.com/payloadcms/payload/releases/tag/v3.25.0
